planning/communication-plan.md

```diff
@@ -28,21 +28,20 @@ how often will we get in touch on each channel, and what we will discuss there:
 
 ### Availability for calling/messaging
 
-| Day     |   Monday    |   Tuesday   |  Wednesday  |  Thursday   |   Friday    |  Saturday   |   Sunday   |
-| ------- | :---------: | :---------: | :---------: | :---------: | :---------: | :---------: | :--------: |
-| _Brian_ | 8am - 12pm  | 8am - 12pm  | 8am - 12pm  | 8am - 12pm  | 8am - 12pm  | 8am - 12pm  | 8am - 12pm |
-| _Inna_  | 15am - 12pm | 15am - 12pm | 15am - 12pm | 9am - 12pm  | 12am - 12pm | 9am - 12pm  |  --------  |
-| _Azima_ | 13am - 21pm | 13am - 21pm | 13am - 21pm | 13am - 21pm | 15pm - 18pm | 10am - 14pm |  --------  |
-| _name_  | 00am - 00pm | 00am - 00pm | 00am - 00pm | 00am - 00pm | 00pm - 00pm | 00am - 00pm |  --------  |
```

Comment:
please run formating commandes before pushing. 
- npm run format
- npm run format:check
- npm run lint:ls
- npm run lint:md
- npm run spell-check

Reply:
I did run these but I'll try it again.